include/robot_localization/ekf.hpp

```diff
@@ -31,53 +31,56 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef ROBOT_LOCALIZATION_EKF_H
-#define ROBOT_LOCALIZATION_EKF_H
+#ifndef ROBOT_LOCALIZATION__EKF_HPP_
+#define ROBOT_LOCALIZATION__EKF_HPP_
 
 #include <robot_localization/filter_base.hpp>
 #include <vector>
 
-
 namespace robot_localization
 {
 
 /**
-  * @brief Extended Kalman filter class
-  *
-  * Implementation of an extended Kalman filter (EKF). This class derives from FilterBase and overrides the predict()
-  * and correct() methods in keeping with the discrete time EKF algorithm.
-  */
-class Ekf: public FilterBase
+ * @brief Extended Kalman filter class
+ *
+ * Implementation of an extended Kalman filter (EKF). This class derives from
+ * FilterBase and overrides the predict() and correct() methods in keeping with
+ * the discrete time EKF algorithm.
+ */
+class Ekf : public FilterBase
 {
-  public:
-    /**
-     * @brief Constructor for the Ekf class
-     */
-    explicit Ekf();
+public:
+  /**
+   * @brief Constructor for the Ekf class
+   */
+  Ekf();
 
-    /**
-     * @brief Destructor for the Ekf class
-     */
-    ~Ekf();
+  /**
+   * @brief Destructor for the Ekf class
+   */
+  ~Ekf();
 
-    /**
-     * @brief Carries out the correct step in the predict/update cycle.
-     *
-     * @param[in] measurement - The measurement to fuse with our estimate
-     */
-    void correct(const Measurement &measurement) override;
+  /**
+   * @brief Carries out the correct step in the predict/update cycle.
+   *
+   * @param[in] measurement - The measurement to fuse with our estimate
+   */
+  void correct(const Measurement & measurement) override;
 
-    /**
-     * @brief Carries out the predict step in the predict/update cycle.
-     *
-     * Projects the state and error matrices forward using a model of the vehicle's motion.
-     *
-     * @param[in] reference_time - The time at which the prediction is being made
-     * @param[in] delta - The time step over which to predict.
-     */
-    void predict(const rclcpp::Time &reference_time, const rclcpp::Duration &delta) override;
+  /**
+   * @brief Carries out the predict step in the predict/update cycle.
+   *
+   * Projects the state and error matrices forward using a model of the
+   * vehicle's motion.
+   *
+   * @param[in] reference_time - The time at which the prediction is being made
+   * @param[in] delta - The time step over which to predict.
+   */
+  void predict(
+    const rclcpp::Time & reference_time,
```

Comment:
Blanket statement for all reference parameters: if we're going to be modifying these, can we make them all of the format

```
const ObjectType& var
```

(Ampersand gets appended to the type). Will the linter complain about that?

Reply:
After following the lint rule on this file like, "clang-format-6.0 -i -style=file include/robot_localization/ekf.hpp"
 output :-1: 
```
-  void predict(
-    const rclcpp::Time & reference_time,
-    const rclcpp::Duration & delta) override;
+  void predict(const rclcpp::Time &reference_time,
+               const rclcpp::Duration &delta) override;
```
So shall we follow this [Time & reference_time] or [Time &reference_time] ? Please suggest.